integrations/hermes/src/mnemosyne_hermes/__init__.py

```
@@ -350,6 +350,13 @@ def _sync_turn_assistant_limit() -> int:
 
 # ---------------------------------------------------------------------------
 from .tools import ALL_TOOL_SCHEMAS
+from mnemosyne.batch_tool import (
+    BatchValidationError,
+    apply_beam_batch,
+    batch_validation_error_payload,
+    dry_run_batch,
+    validate_batch_operations,
+)
```

Ruff (0.15.20)

[error] Module level import not at top of file

(E402)